src/utils/file/index.ts

```diff
@@ -333,9 +334,7 @@ export function generateStreamFromArrayBuffer(data: Uint8Array) {
 
 export async function convertForPreview(file: EnteFile, fileBlob: Blob) {
     if (file.metadata.fileType === FILE_TYPE.LIVE_PHOTO) {
-        const originalName = fileNameWithoutExtension(file.metadata.title);
-        const motionPhoto = await decodeMotionPhoto(fileBlob, originalName);
-        fileBlob = new Blob([motionPhoto.image]);
+        return fileBlob;
```

Comment:
It would be better if you transcode the live photo assets  here itself and then return an array of file blobs [ imageBlob, videoBlob] 

returning the `fileBlob` without conversion can later cause confusions